### PR TITLE
#54: Prepared release 5.3.0

### PR DESCRIPTION
**Pull request by tomuben**
Prepared first official release of r-exasol.
Also added yml files needed by release-droid.

### CODE REVIEW

File: doc/changes/changes_5.3.0.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 
-t.b.d.
+This is the first official release of r-exasol. 
 
 ## Features / Enhancements
 * Initial implementation
 * #41: Added connection pane snippets (#57)
+* #61: add standard documentation (#62)
+* #59: Move CI build system from Travis to Github actions (#60)
+* #65: Log errno when socket access fails (#66)
```
Comment (morazow) [inline doc/changes/changes_5.3.0.md:14]:
```suggestion
* #61: Added standard documentation (#62)
* #59: Migrated CI build system from Travis to Github actions (#60)
* #65: Added logging errno when socket access fails (#66)
```

File: .github/workflows/release_droid_prepare_original_checksum.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        run: echo "$SECRET_DOCKER_TOKEN" | docker login --username "$SECRET_DOCKER_USER_NAME" --password-stdin
+        env: # Set the secret as an input
+          SECRET_DOCKER_USER_NAME: ${{ secrets.DOCKER_USER_NAME }}
+          SECRET_DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      - name: Run all tests
+        run: |
+          ./tests/scripts/pull_docker_image.sh ${{ matrix.r-version }}
+          ./tests/scripts/execute_docker_test_env.sh ${{ matrix.r-version }}
+
+  publish-main:
```
Comment (morazow) [inline .github/workflows/release_droid_prepare_original_checksum.yml:48]: The main CI gihub workflow also has a publish step, do we need it there still?

**Review by morazow — APPROVED** [review]
Looks good, added minor edits to changes file.

File: .github/workflows/release_droid_prepare_original_checksum.yml (hunk 2 above)
Comment (tomuben) [reply .github/workflows/release_droid_prepare_original_checksum.yml:48]: Yes, this step pushes a new docker image to docker hub (which then is used by the CI build itself).

**Review by morazow — APPROVED** [review]

**Pull request merged**